lib/runner.js

```diff
@@ -302,7 +302,13 @@ Runner.prototype.hook = function(name, fn) {
       }
       if (err) {
         if (err instanceof Pending) {
-          suite.pending = true;
+          if (name === 'beforeEach' || name === 'afterEach') {
+            self.test.pending = true;
+          } else {
+            suite.tests.forEach(function(test) {
+              test.pending = true;
+            });
```

Comment:
Couldn't it use `suite.pending` if it's not `beforeEach` or `afterEach`? Or in the older versions does `after` run even if `before` calls this.skip()?